temperature/models.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, ForeignKey, DateTime, Float
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "Temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer, ForeignKey("City.id"), nullable=False)
```

Comment:
Verify that the `City` model is correctly defined and that the `ForeignKey` reference to `City.id` is valid. This ensures referential integrity between the `Temperature` and `City` tables.